spec/takeaway_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'takeaway'
+require 'menu'
+require 'order'
+
+describe Takeaway do
+
+  let(:order) { Order.new }
+  let(:printed_menu) { {
+
+    'Salmon Poke Bowl' => 8,
+    'Tuna Poke Bowl' => 8,
+    'Tofu Poke Bowl' => 8,
+    'Chicken Poke Bowl' => 8,
+
+    'Blue Hawaii Cocktail' => 7,
+    'Mai Tai Cocktail' => 7,
+    'Hawaiian Margarita Cocktail' => 7,
+    'Mango Martini Cocktail' => 7
+
+    }
+  }
+
+  it 'creates a new instance of takeaway' do
+    expect(subject).to be_instance_of Takeaway
+  end
+
```

Comment:
using the double you have created :printed menu, the next test could be  

it "returns the menu of dishes and prices" do
    expect(takeaway.printed_menu).to eq(view_menu)
  end